<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>696</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1261</width>
    <height>696</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1261</width>
    <height>696</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1261</width>
     <height>696</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1261</width>
     <height>696</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1261</width>
     <height>696</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">.QStackedWidget {
    border-style: solid;
    border-color: rgb(186, 189, 182);
    border-width: 1px;
    border-radius: 6px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btnNewProgram">
     <property name="geometry">
      <rect>
       <x>965</x>
       <y>50</y>
       <width>171</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Create New</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>861</width>
       <height>421</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>859</width>
        <height>419</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>45</y>
         <width>246</width>
         <height>46</height>
        </rect>
       </property>
       <property name="text">
        <string>List Of programs here</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>10</y>
       <width>1066</width>
       <height>86</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
    color: rgb(238, 238, 236);
	font: 14pt &quot;Bebas Kai&quot;;
	border-style: solid;
    border-color: rgb(186, 189, 182);
    border-width: 1px;
    border-radius: 6px;
}</string>
     </property>
     <property name="title">
      <string>Program Header</string>
     </property>
     <widget class="DialogButton" name="btnHeader">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>25</y>
        <width>126</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Header</string>
      </property>
      <property name="dialogName" stdset="0">
       <string>program_header</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEditHeader">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>25</y>
        <width>126</width>
        <height>46</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Header</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>135</y>
       <width>1066</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
    color: rgb(238, 238, 236);
	font: 14pt &quot;Bebas Kai&quot;;
	border-style: solid;
    border-color: rgb(186, 189, 182);
    border-width: 1px;
    border-radius: 6px;
}</string>
     </property>
     <property name="title">
      <string>Program Operations</string>
     </property>
     <widget class="DialogButton" name="btnAddOperation">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>126</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Operation</string>
      </property>
      <property name="dialogName" stdset="0">
       <string>facing_operation</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>675</x>
       <y>590</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Code &amp;&amp; Load Program</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBackToPrograms">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>126</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Back To
Programs</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DialogButton</class>
   <extends>QPushButton</extends>
   <header>qtpyvcp.widgets.button_widgets.dialog_button</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
